lib/livebook_cli/server.ex

```diff
@@ -126,6 +127,10 @@ defmodule LivebookCLI.Server do
       |> append_path("/explore/notebooks/new")
       |> Livebook.Utils.browser_open()
     end
+    
+    if Keyword.has_key?(opts, :notebook) do
+      LivebookApp.import_livebook("file://" <> File.cwd!() <> Keyword.fetch!(opts, :notebook))
```

Comment:
Ah there's a catch! We compile the `LivebookApp` module conditionally if mix target is `:app`, so we shouldn't rely on this. Instead we can move the URL computation somewhere else, I'm thinking `LivebookWeb.Helpers.notebook_import_url` :)

Reply:
I'd make the function to return the URL without opening it, as that's more decoupled :)

Reply:
got it :)

Reply:
made some edits. thanks for walking me through this :)